scripts/install_moodle.sh

```diff
@@ -883,6 +883,9 @@ EOF
    #else # mssql. TODO It's missed earlier! Complete this!
    fi
 
+   # setting file locking to false
+   sed -i "23 a \$CFG->preventfilelocking = false;" /moodle/html/moodle/config.php
```

Comment:
Could you please explain what this change does? I did a quick search on this, but I didn't get anything useful.

Also, can we move this line to another place where the `/moodle/html/moodle/config.php` file is actively updated, like line 861? We should try to group all related code together, not scatter them around.